app/views/events/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <h1 class="text-center pt-3 pb-2">Edit Event</h1>
+
+  <% if @event.errors.any? %>
+    <div id="error_explanation" class="mt-2">
+      <h2><span class="text-danger"><%= pluralize(@event.errors.count, "error") %></span> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% @event.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ] Since this method has more than one line of markup in the condition, create a method that renders a partial that contains all the markup within the if statement.